pkg/alpine/go-compile.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Where are you planning on using this go-compile script?

Reply:
It is actually used already in that 2nd commit https://github.com/lf-edge/eve/pull/1969/files#diff-99172ad88d990c8dd2aab7cae44da2b8af22fade6589ee45d39812339526c4b6R11

IOW, it is there so our existing containers don't break (note that I didn't add the use of it to rngd container -- it has always been there @eriknordmark )